content/project/team-leads.adoc

```diff
@@ -1,12 +1,44 @@
 ---
 layout: simplepage
-title: "Team leads"
+title: "Jenkins Team Leads and Officers"
 section: project
 ---
 
-## Security
+Jenkins team leads and officers are individual contributors have the responsibility and accountability for
+leading a particular facet of the Jenkins project and/or the community.
+These roles are held by community members who have proven themselves to be responsible and courteous,
+representing the executive branch of the Jenkins project governance model.
+Team leads and officers act with the authority in guiding and making decisions within the project, and representing the Jenkins project within the area of their responibility.
+Officer roles are an essential part of Jenkins' community governance and well-being.
 
-Coordinates the security-related activities in the Jenkins project.
+The difference between _Jenkins Officers_ and _Team Leads_ lays in the appointement process:
+
+* Team Leads are appointed by the link:/project/governance-meeting/[Jenkins Governance Meeting] after nomination in the Jenkins developer mailing list.
+  The Jenkins team leads do not have a specific term, roles may be introduced and transferred as deemed necessary by the community.
+* Jenkins Officer is an *elected role*.
+  Officers are elected by the Jenkins community during the regular link:/project/board-election-process/[Jenkins elections].
+  Jenkins Officers have a fixed term though the number of terms is not limited.
+
+## Responsibilities
+
+All Jenkins officers and team leads have the following responsibilities:
+
+* Coordinating the work inside the teams and SIGs they lead.
+  While team leads are responsible to ensure that the mission-critical community processes work, they are not expected to work on them single-handedly.
+* Onboarding new contributors to the teams they lead.
+* Mentorship and knowledge sharing within the team to ensure healthy officer rotation in the project,
+  with opportunity for team leads to move on to other roles or to take a break when needed.
+* Facilitating initiatives within the scope of the responsibility, maintaining the team's roadmap,
+  and ensuring that key team initiatives reflected on the link:/project/roadmap/[public Jenkins roadmap].
+* Representing the Jenkins project and community within the area of their responsibility.
+
+
+## Team Leads and Officers
+
+[[security]]
+### Security Officer
+
+Coordinates the security-related activities in the Jenkins project. This is an **elected role**. 
 
 * Run the Jenkins CERT meeting
 * link:https://wiki.jenkins.io/display/JENKINS/Rewards+for+reporting+security+issues[Manage sending gifts to qualifying reporters of resolved security issues]
```

Comment:
```suggestion
* link:https://www.jenkins.io/security/gift/[Manage sending gifts to qualifying reporters of resolved security issues]
```

Reply:
Also line 46 should link to https://www.jenkins.io/security/advisories/

Reply:
Probably should be removed at all. 
I would rather refer "Maintain the vulnerability reporting process" or so on.

Reply:
Probably.

Reply:
```suggestion
* link:/security/gift/[Manage sending gifts to qualifying reporters of resolved security issues]
```

Will copy-edit later